docs/markdown/Contributions/releases/release-process.md

```diff
@@ -94,7 +94,7 @@ Update the release page in `src/python/pants/notes` for this release series, e.g
 
 Run `git fetch --all --tags` to be sure you have the latest release tags available locally.
 
-From the `main` branch, run `pants run build-support/bin/changelog.py -- --prior 2.9.0.dev0 --new 2.9.0.dev1` with the relevant versions. 
+From the `main` branch, run `./pants run build-support/bin/changelog.py -- --prior 2.9.0.dev0 --new 2.9.0.dev1` with the relevant versions.
```

Comment:
Why this change?

Reply:
I think `./pants` is still the officially sanctioned way to run Pants in the pantsbuild repo.